README.md

```diff
@@ -81,6 +81,85 @@ You will also need:
 
 * An Okta account, called an _organization_ (sign up for a free [developer organization](https://developer.okta.com/signup) if you need one). 
 
+## Usage guide
+
+These examples will help you understand how to use this library.
+
+### Get Proceed Context
+
+```java
+ProceedContext proceedContext = Util.getProceedContextFromSession(session);
```

Comment:
This isn't an SDK method, it's only in the sample. Do we want to list it here?

Reply:
ah good point, will remove.

Reply:
done